src/App.js

```diff
@@ -1,20 +1,64 @@
+import { useEffect, useState } from 'react'
+import { v4 as uuidv4 } from 'uuid'
 import styled from 'styled-components'
+
+import FloatingActionButton from './components/FloatingActionButton'
+
+import loadLocally from './lib/loadLocally'
+import AddBookForm from './components/AddBookForm'
+import getBooks from './services/getBooks'
+
 import BookList from './components/BookList'
+import BookCard from './components/BookCard'
 
 export default function App() {
+  const [books, setBooks] = useState(loadLocally('books') ?? [])
+
+  const [addBookFormModal, setaddBookFormModal] = useState(false)
+  function closeAddBookFormModal() {
+    setaddBookFormModal(false)
+  }
+  function openAddBookFormModal() {
+    setaddBookFormModal(true)
+  }
+
+  useEffect(() => {
+    getBooks().then((books) => setBooks(books))
+  }, [])
+
   return (
     <StyledApp>
       <h1>Book Owls</h1>
       <BookList listName="Bücherregal" />
+      <FloatingActionButton onClick={openAddBookFormModal} />
+      {addBookFormModal && (
+        <AddBookForm
+          onCreateBook={addBook}
+          onClickDownButton={closeAddBookFormModal}
+        />
+      )}
+      {books.map(({ title, author, description, id }, index) => (
```

Comment:
Is it intentional that the books you add by yourself are not added to the BookList component? I added a book and it wasn't rendered within the booklist, which makes it look a bit different than the other books, for example there is no padding. 

I would suggest to add `books` to your booklist component as props. It is a bit odd to have `BookCard`s rendered at two different places. If you have a question, please let me know.